python/app/dialog.py

```diff
@@ -374,6 +377,9 @@ def focus_entity(self):
         curr_index = self.ui.entity_tab_widget.currentIndex()
 
         if self._current_entity_tabs[curr_index] == self._current_location.tab:
+            # FIXME: It appears due to the changes above that we are unlikely to hit this point in the code.
+            #        Since the tabs are rebuilt each time the current index is
+            #        never going to match unless you are on the first tab.
```

Comment:
Is this something you are going to address in this PR? If not, can you create a Jira ticket about it, please? And then add the ticket ID to the comment.

Reply:
Just something I noticed which debugging the code, not something I need to address but a warning for anyone else poking around there. I can put a ticket in though it looks like it should be address in the QT update based on the comments above, maybe it should be linked to what ever work is happening there?